src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -3237,11 +3282,116 @@ public void RecordStructParsing_WrongOrder()
             EOF();
         }
 
+        [Fact, CompilerTrait(CompilerFeature.RecordStructs), WorkItem(59534, "https://github.com/dotnet/roslyn/issues/59534")]
+        public void StructNamedRecord_CSharp8()
```

Comment:
>StructNamedRecord_CSharp8 [](http://example.com/codeflow?start=20&length=25)

Consider merging these three into a single test, either with `[Theory]` or a single `verify(ParseOptions)` method, and add a similar test for `class record { }`.

Reply:
We have an IDE PR waiting for this to be merged, so I'd rather avoid one more iteration to combine two tests. Thanks